components/ItemsPage/AddOns/AddOns.tsx

```diff
@@ -6,6 +6,7 @@ import useElementRefList from '../../../hooks/useElementRefList'
 import { useEffect, useState } from 'react'
```

Comment:
Keep in mind that this file will conflict with PR #113 (Settings to Item Page). So go ahead and decide which pr to merge first, then solve the conflict. But I highly recommend you to merge first the Order Page fix then you decide about the other 2.